File: csharp/DVSClient/DVSClient/server/address/layout/RestApiGetLayoutListResponse.cs
using Newtonsoft.Json;

namespace DVSClient.Server.Address.Layout
{
    public class RestApiGetLayoutListResponse : RestApiResponse
    {
        [JsonProperty("error")]
        public RestApiResponseError? Error { get; set; }

        [JsonProperty("result")]
        public IEnumerable<RestApiGetLayoutsListItem>? Result { get; set; }
    }
}

File: typescript/src/server/address/layout/restApiGetLayoutListResponse.ts
import { RestApiResponse } from "../../restApiResponse";
import { RestApiResponseError } from "../../restApiResponseError";
import { RestApiGetLayoutsListItem } from "./restApiGetLayoutsListItem";

export type RestApiGetLayoutListResponse = RestApiResponse & {
    error?: RestApiResponseError;
    result?: RestApiGetLayoutsListItem[];
};